styles/styles.css

```diff
@@ -1,41 +1,103 @@
+@import url('./fonts.css');
+
 * {
   margin: 0;
   /*box-sizing: border-box;*/
 }
 
-.container {
-  width: 1440px;
+.body {
+  font-family: 'Roboto', sans-serif;
+}
+
+.container-advantage {
+  max-width: 1440px;
+  height: 449px;
```

Comment:
Look at the example in the [article about padding](https://www.w3schools.com/css/css_padding.asp) to understand how to avoid the calculations of height/width.